macros/Compat.ejs

```diff
@@ -99,19 +116,19 @@ a string to appear in the table cell, like "Yes", "No" or "?"
 function getVersionString(versionInfo) {
   switch (versionInfo) {
     case null:
-      return `<span title="${localize(compatStrings, 'supportsShort_unknown_title')}"
-              style="color: rgb(255, 153, 0);">${localize(compatStrings, 'supportsShort_unknown')}</span>`;
-      break;
+      return {class: 'unknown-support', text: `<span title="${localize(compatStrings, 'supportsShort_unknown_title')}"
```

Comment:
Yes, I originally backported this from the webext compat macro, but you are right: The whole business of getting a CSS class is actually not straight forward and is now in an own function `getSupportClass`.